### PR TITLE
Etherpad Logic Phase 1

### PR DESCRIPTION
**Pull request by anantjain6**
In continuation to the conversation of #6

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -51,4 +51,4 @@ script:
   
   # Run PHP Unit Test
   - cd core/
-  - ../vendor/bin/phpunit --group collaborative_editors
+  - ../vendor/bin/phpunit --group collaborative_editors
```
Comment (ankitjain28may) [inline .travis.yml]: Add new line at the end

File: modules/ce_etherpad/ce_etherpad.libraries.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  version: VERSION
+  js:
+    js/editor.js: {}
+  css:
+    component:
+      styles/editor.css: {}
+  dependencies:
+    - core/jquery
+    - core/drupal
+    - core/drupalSettings
```
Comment (ankitjain28may) [inline modules/ce_etherpad/ce_etherpad.libraries.yml]: Add new line at the end

File: modules/ce_etherpad/js/editor.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+(function (Drupal) {
+  var ce_etherpad = {
+    attach: function attach(element, format) {},
+    detach : function detach(element, format, trigger){},
+    onChange: function onChange(element, callback) {}
+  };
+  Drupal.editors.ce_etherpad = ce_etherpad;
+})(Drupal);
```
Comment (ankitjain28may) [inline modules/ce_etherpad/js/editor.js]: Add new line at the end

**Review by ankitjain28may — CHANGES_REQUESTED** [review]

File: modules/ce_etherpad/src/Plugin/CollaborativeNetwork/EtherpadEditor.php [hunk 4]
```diff
@@ -50,7 +50,7 @@ public function setCredentials($apiKey, $apiUrl = 'http://localhost:9001') {
    */
   public function testConnection() {
     try {
-      $client = new Client($this->apiKey, $this->apiUrl);
+      $client = new Client($this->apiUrl, $this->apiKey);
```
Comment (anantjain6) [inline modules/ce_etherpad/src/Plugin/CollaborativeNetwork/EtherpadEditor.php]: This change was a mistake

**Comment by anantjain6** [comment]
Suggestions implemented with commit d531577

**Comment by anantjain6** [comment]
@ankitjain28may Plz merge the PR

**Review by ankitjain28may — APPROVED** [review]

**Pull request merged**